tests/test_kldiv.py

```diff
@@ -145,5 +145,15 @@ def test_oned(session_tf, white, mu, sqrt, K_batch):
     np.testing.assert_allclose(kl.eval(), kl_tf.eval())
 
 
+def test_unknown_size_inputs(session_tf):
```

Comment:
it would be nice to have docstring describing the issue and reference to github issue